apps/test/unit/lib/levelbuilder/code-docs-editor/ImageInputTest.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import sinon from 'sinon';
+import ImageInput from '@cdo/apps/lib/levelbuilder/code-docs-editor/ImageInput';
+
+describe('ImageInput', () => {
```

Comment:
The previous filename was `ImageInputText` so I renamed it. Unfortunately, there were enough changes that github thinks this file is brand new.

Reply:
thank you for pointing this out! I find that the only way to make this easily reviewable is to do the rename in a single commit, and then do the changes in another commit. even so, github PR view won't show you that the file is moved, but at least then the reviewer can go through the commits and see all the substantial changes in a certain commit or range of commits. no need to go back and to it here -- I can always diff these locally if needed

Reply:
Good point -- I was hopeful that git would be clever but it was not. I copied this work over to another branch and 25bbbb5895e3b99bff39a81e91d2cf729d70e5eb is the actual work. I'm not sure if it's better to close this PR in favor of one on that branch -- curious to hear what you think

Reply:
I can review this work given all of this -- thanks! not super important to me which gets merged.

Reply:
Great! I'll leave this one and close the other so it's not confusing. The commit should still be around even after I close that PR, so it can still be used as a reference